<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuWidget</class>
 <widget class="QWidget" name="MenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>576</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>358</width>
    <height>357</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>579</width>
    <height>576</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>商品浏览</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
border-radius:8px;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>current account:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="usernameLabel">
         <property name="text">
          <string>haha123</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>out</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="myOrderButton">
       <property name="text">
        <string>my deals</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="releaseButton">
       <property name="text">
        <string>upload product</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="myinfoButton">
       <property name="text">
        <string>personal info</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="table">
     <property name="textElideMode">
      <enum>Qt::TextElideMode::ElideRight</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>product num</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="buyIdEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buyButton">
       <property name="text">
        <string>buy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QComboBox" name="speciesComboBox">
       <item>
        <property name="text">
         <string>number</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>price</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="howComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>increasing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>decreasing</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="orderButton">
       <property name="text">
        <string>sort</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>product name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchIdEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="text">
        <string>search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RenewpushButton">
       <property name="text">
        <string>reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
